srv/src/main/resources/messages_de.properties

```diff
@@ -10,4 +10,5 @@ review.invalid.rating = Die Bewertung muss ein Wert innerhalb 1 und 5 sein
 review.added = Bewertung hinzugef\u00FCgt
 order.exceeds.stock = {0} ist mehr als f\u00FCr das Buch auf Vorrat \u00FCbrig ist
 review.add.forbidden=Es ist nicht mehr als eine Bewertung pro Buch erlaubt
-book.import.failed = B\u00FCcher Import fehlgeschlagen
\ No newline at end of file
+book.import.failed = B\u00FCcher Import fehlgeschlagen
+book.import.invalid.csv = Die CSV Datei enthält eine ungültige Struktur - Bitte \u00FCberpr\u00FCfen Sie den Inhalt
```

Comment:
Do we still have issues with umlauts? The fact that you only replaced half of the umlauts maybe suggests that it is not required anymore to do this?

Reply:
No, I just forgot to change it when I tried out different phrases. I have not tested with German locale, I just assumed it works as suggested by the existing entries.

Reply:
Okay :) I was always wondering if this is still necessary and just tested it and it is not. I think we fixed that bug long ago, that we couldn't handle umlauts there. I will open a separate PR. You can use normal umlauts for your new error texts.

Reply:
okay :)

Reply:
https://github.com/SAP-samples/cloud-cap-samples-java/pull/93